### ISSUE TITLE
Adding Simple-Data-Grapher button in tabs and Per-User-Data button on profile page

### ISSUE THREAD

**Issue by IshaGupta18** [publiclab/plots2#6228]
Now that Simple-Data-Grapher is integrated in plots2, we need to find an appropriate place to display the feature for use.

Maybe we can put it here, under "Post your work" in the "Get Involved" tab
or we can put it under the "Share your work" button on the right side?
![image](https://user-images.githubusercontent.com/40794215/64072143-46be6b80-cca7-11e9-842e-b865ec76bcb5.png)

About the per-user data page, it currently opens on 'graph/data/:uid'. We can insert it somewhere in the profile page, probably in the grey texts under the profile picture or in the tabs on hovering over the little profile picture?

![image](https://user-images.githubusercontent.com/40794215/64072177-ee3b9e00-cca7-11e9-89c5-d508afb613cd.png)

Let's decide the optimal place for this, so that this can be done and the viewers can start using SDG!

@ebarry @CleverFool77 @gautamig54 @jywarren please suggest! Thank you!

**Comment by gautamig54** [comment]
Hi @IshaGupta18! Great work with the integration. 
Regarding placing it on the dashboard, we have to wait for @jywarren because in the new design of dashboard, we are getting rid of the share your work dropdown. Refer #6072 
We can place it the main user dropdown on the navbar. On the profile page, we can add it as one of the options on the toggle ellipsis on the sidebar.

**Comment by IshaGupta18** [comment]
Sounds good, but toggle ellipsis might hide it, where else do you think it
will look good on the profile page?

**Comment by gautamig54** [comment]
As per the new design, I am not sure whee else we can place it. Let's wait for @jywarren's view on this.

**Comment by IshaGupta18** [comment]
Okay, thank you!

**Comment by jywarren** [comment]
Hi all! In our #5211 perhaps we can have a type in the popup modal... https://github.com/publiclab/plots2/pull/6114

But i think it's important that there be some more context -- like, why, what kind of data? 

The other thing we can do, is that if we have a really good step by step activity with screenshots, we can post it on https://publiclab.org/data-logging and other similar pages throughout the site, which is done in content, rather than in code. That would mean posting an activity and marking it with `activity:data-logging` and other appropriate tags.

**Comment by jywarren** [comment]
That would give a really good space to offer more context and explanation!

**Comment by IshaGupta18** [comment]
Hey I am not able to follow very clearly. Can you please explain? What do
we have to do with the SDG button?

**Comment by ebarry** [comment]
I'm so excited to see this integrated. What plans do we have for featuring our previous platforms like mapknitter.org and spectralworkbench.org via navigation on publiclab.org? Could we do something consistent with the Data Grapher?

**Comment by IshaGupta18** [comment]
Yeah sure, maybe we can integrate it like plots2 in mk and swb?

**Comment by ebarry** [comment]
I don't think we have ever announced or prominently linked to this feature -- let's figure out how to promote this! By commenting, i have removed the 'stale' label.
